packages/tempus/src/Tempus_StepperOperatorSplit_impl.hpp

```diff
@@ -59,6 +62,43 @@ StepperOperatorSplit<Scalar>::StepperOperatorSplit(
   this->setOrderMax(orderMax);
 
   this->setObserver(obs);
+  this->setAppAction(Teuchos::null);
+
+  OpSpSolnHistory_ = rcp(new SolutionHistory<Scalar>());
+  OpSpSolnHistory_->setStorageLimit(2);
+  OpSpSolnHistory_->setStorageType(Tempus::STORAGE_TYPE_STATIC);
+
+  if ( !(appModels.empty()) ) {
+    this->setModels(appModels);
+    this->initialize();
+  }
+}
+#endif
+
+template<class Scalar>
+StepperOperatorSplit<Scalar>::StepperOperatorSplit(
+  std::vector<Teuchos::RCP<const Thyra::ModelEvaluator<Scalar> > > appModels,
+  std::vector<Teuchos::RCP<Stepper<Scalar> > > subStepperList,
+  const Teuchos::RCP<StepperOperatorSplitAppAction<Scalar> >& stepperOSAppAction,
+  bool useFSAL,
+  std::string ICConsistency,
+  bool ICConsistencyCheck,
+  int order,
+  int orderMin,
+  int orderMax)
+{
+  this->setStepperType(        "Operator Split");
+  this->setUseFSAL(            useFSAL);
+  this->setICConsistency(      ICConsistency);
+  this->setICConsistencyCheck( ICConsistencyCheck);
+
+  this->setSubStepperList(subStepperList);
+  this->setOrder   (order);
+  this->setOrderMin(orderMin);
+  this->setOrderMax(orderMax);
+
+  this->setObserver();
```

Comment:
Need a TEMPUS_HIDE_DREPRECATED_CODE around the setObserver.

Reply:
Fixed.